### ISSUE TITLE
ftgogo docker-compose up build error

### ISSUE THREAD

**Issue by c-nv-s** [stackus/edat#1]
```
# COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker-compose up

...

Building delivery-service
[+] Building 7.2s (23/24)                                                                                                                                                                                  
 => [internal] load build definition from Dockerfile                                                                                                                                                  0.4s
 => => transferring dockerfile: 32B                                                                                                                                                                   0.0s
 => [internal] load .dockerignore                                                                                                                                                                     0.6s
 => => transferring context: 34B                                                                                                                                                                      0.0s
 => [internal] load metadata for docker.io/library/golang:alpine                                                                                                                                      2.3s
 => [internal] load metadata for docker.io/library/alpine:latest                                                                                                                                      2.3s
 => [builder  1/17] FROM docker.io/library/golang:alpine@sha256:b683b85530b81e84a1365014a4c2a8850409e9d1cfd199b06f13624bbcf7cb08                                                                      0.0s
 => [internal] load build context                                                                                                                                                                     0.6s
 => => transferring context: 5.30kB                                                                                                                                                                   0.0s
 => [runtime 1/2] FROM docker.io/library/alpine:latest@sha256:234cb88d3020898631af0ccbbcca9a66ae7306ecd30c9720690858c1b007d2a0                                                                        0.0s
 => CACHED [builder  2/17] RUN mkdir /go/src/shared-go                                                                                                                                                0.0s
 => CACHED [builder  3/17] COPY ./shared-go/go.* /go/src/shared-go                                                                                                                                    0.0s
 => CACHED [builder  4/17] WORKDIR /go/src/shared-go                                                                                                                                                  0.0s
 => CACHED [builder  5/17] RUN go mod download                                                                                                                                                        0.0s
 => CACHED [builder  6/17] RUN mkdir /go/src/serviceapis                                                                                                                                              0.0s
 => CACHED [builder  7/17] COPY serviceapis/go.* /go/src/serviceapis                                                                                                                                  0.0s
 => CACHED [builder  8/17] WORKDIR /go/src/serviceapis                                                                                                                                                0.0s
 => CACHED [builder  9/17] RUN go mod download                                                                                                                                                        0.0s
 => CACHED [builder 10/17] RUN mkdir /go/src/delivery                                                                                                                                                 0.0s
 => CACHED [builder 11/17] COPY ./delivery/go.* /go/src/delivery                                                                                                                                      0.0s
 => CACHED [builder 12/17] WORKDIR /go/src/delivery/cmd/service                                                                                                                                       0.0s
 => CACHED [builder 13/17] RUN go mod download                                                                                                                                                        0.0s
 => CACHED [builder 14/17] COPY serviceapis /go/src/serviceapis                                                                                                                                       0.0s
 => CACHED [builder 15/17] COPY ./shared-go /go/src/shared-go                                                                                                                                         0.0s
 => CACHED [builder 16/17] COPY ./delivery /go/src/delivery                                                                                                                                           0.0s
 => ERROR [builder 17/17] RUN go install .                                                                                                                                                            3.4s
------                                                                                                                                                                                                     
 > [builder 17/17] RUN go install .:                                                                                                                                                                       
#23 1.672 go: github.com/stackus/edat@v0.0.3: missing go.sum entry; to add it:
#23 1.672       go mod download github.com/stackus/edat
------
executor failed running [/bin/sh -c go install .]: exit code: 1
ERROR: Service 'delivery-service' failed to build : Build failed
```

**Comment by stackus** [comment]
Sorry about that.

I'll push up the missing file asap, in the mean time to generate the missing file use the `mod tidy` command in the delivery directory.

```shell
cd delivery
go mod tidy
```

**Comment by c-nv-s** [comment]
Also just fyi
https://github.com/looplab/eventhorizon
you may have found that library useful

**Comment by stackus** [comment]
Build issues were resolved by PR stackus/ftgogo#2 and the services should all now use v0.0.6.
